tests/foreman/longrun/test_provisioning_computeresource.py

```diff
@@ -26,68 +28,100 @@
 from robottelo.utils.issue_handlers import is_open
 
 
-@pytest.fixture(scope="module")
-def bridge():
-    return settings.vlan_networking.bridge
-
-
-@pytest.fixture(scope="module")
-def rhev(bridge):
-    rhev = {}
-    rhev['rhev_url'] = settings.rhev.hostname
-    rhev['rhev_password'] = settings.rhev.password
-    rhev['rhev_username'] = settings.rhev.username
-    rhev['rhev_datacenter'] = settings.rhev.datacenter
-    rhev['rhev_img_name'] = settings.rhev.image_name
-    rhev['rhev_img_arch'] = settings.rhev.image_arch
-    rhev['rhev_img_os'] = settings.rhev.image_os
-    rhev['rhev_img_user'] = settings.rhev.image_username
-    rhev['rhev_img_pass'] = settings.rhev.image_password
-    rhev['rhev_vm_name'] = settings.rhev.vm_name
-    rhev['rhev_storage_domain'] = settings.rhev.storage_domain
-    rhev['rhv_api'] = RHEVMSystem(
-        hostname=rhev['rhev_url'].split('/')[2],
-        username=rhev['rhev_username'],
-        password=rhev['rhev_password'],
+@pytest.fixture(scope="function")
+def rhev():
+    bridge = settings.vlan_networking.bridge
+    rhev = namedtuple(
```

Comment:
Thanks for taking the time to make this switch so we have attributized access! 

There is a bit more that needs done here - using `namedtuple` is kind of a two step process.

First, an instance of `namedtuple` is created, which acts like a class definition.  Then that is used to create individual instances of that type.

```
RHEVSettings = namedtuple('RHEVSettings', [<field names>])
rhev = RHEVSettings(settings.rhev.hostname, settings.rhev.password, <etc>)
```

Any thoughts on ways to simplify these blocks, so that we're not having to map all of these fields?